molo/core/tests/test_commands.py

```diff
@@ -55,6 +63,62 @@ def test_add_language_to_pages(self):
         self.assertEquals(article.language, english)
         self.assertEquals(yourmind.language, english)
 
+    def test_add_translated_pages_to_pages(self):
+        # create article in english with translation in spanish and french
+        english_article = self.article
+        spanish_article = self.mk_article_translation(
+            english_article, self.spanish)
+        french_article = self.mk_article_translation(
+            english_article, self.french)
+
+        # create section in english with translations
+        english_section = self.yourmind
+        spanish_section = self.mk_section_translation(
+            self.yourmind, self.spanish)
+        french_section = self.mk_section_translation(
+            self.yourmind, self.french)
+
+        self.assertFalse(english_article.translated_pages.exists())
+        self.assertFalse(spanish_article.translated_pages.exists())
+        self.assertFalse(french_article.translated_pages.exists())
+        self.assertFalse(english_section.translated_pages.exists())
+        self.assertFalse(spanish_section.translated_pages.exists())
+        self.assertFalse(french_section.translated_pages.exists())
+
+        # run command to add languages to these pages
+        call_command('add_language_to_pages')
+
+        # run command to add list of translated_pages to these pages
+        call_command('add_translated_pages_to_pages')
+
+        # test translated pages for articles
+        self.assertTrue(english_article.translated_pages.filter(
+            pk=spanish_article.pk).exists())
+        self.assertTrue(english_article.translated_pages.filter(
+            pk=french_article.pk).exists())
+        self.assertTrue(spanish_article.translated_pages.filter(
+            pk=english_article.pk).exists())
+        self.assertTrue(spanish_article.translated_pages.filter(
+            pk=french_article.pk).exists())
+        self.assertTrue(french_article.translated_pages.filter(
+            pk=english_article.pk).exists())
+        self.assertTrue(french_article.translated_pages.filter(
+            pk=spanish_article.pk).exists())
```

Comment:
Please add checks that the pages don't contain themselves (and for the sections below too)